root/static/scripts/edit/URLCleanup.js

```diff
@@ -1540,6 +1540,13 @@ const CLEANUPS = {
       return /^https:\/\/musopen\.org\/music\/\d+\/$/.test(url);
     },
   },
+  'muziekweb': {
+    match: [new RegExp('^(https?://)?www\\.muziekweb\\.eu/Link/', 'i')]
```

Comment:
Any reason to auto-select relationship type for URLs with path starting with `/Link/` only?
That might be an issue to enter `https://www.muziekweb.eu/en/Link/JK239095/Alientronic`.
```suggestion
    match: [new RegExp('^(https?://)?www\\.muziekweb\\.eu/', 'i')]
```